static/cfg-view.js

```diff
@@ -91,6 +94,13 @@ function Cfg(hub, container, state) {
         this.networkOpts.interaction.navigationButtons = !this.networkOpts.interaction.navigationButtons;
         this.cfgVisualiser.setOptions(this.networkOpts);
     }, this));
+    this.domRoot.find('.enable-physics-btn').on('click', _.bind(function () {
+        this.networkOpts.physics.enabled = !this.networkOpts.physics.enabled;
+        //change only physics.enabled option to preserve current node locations
+        this.cfgVisualiser.setOptions({
```

Comment:
Could you factor this out and create an `updateOptions' that simply does `this.cfgVisualiser.setOptions(this.networkOpts);` 
Replace the line 95 with a call to this new function too

Reply:
That's what I initially wanted to do, but it can't be applied here, read the comment. Setting all options causes cfgVisualiser layout to be reinitialized loosing positions obtained by dragging nodes manually or physics simulation.

Reply:
You're right! Sorry about this :)